packages/snjs/lib/services/api/session_manager.ts

```diff
@@ -120,6 +120,11 @@ export class SNSessionManager extends PureService<SessionEvent> {
     }
   }
 
+  public getPasswordCreatedDate(): number | undefined {
+    const rootKey = this.protocolService.getRootKey()
+    return rootKey ? Number(rootKey.keyParams.content.created) : undefined;
```

Comment:
```suggestion
    return rootKey ? rootKey.keyParams.createdDate : undefined;
```

Reply:
This would change the return type to date. Do you need it to be a Number?

Reply:
No, I just didn't want stringified timestamp, therefore converted to Number :) Thanks, updated!